src/cmdlib.py

```diff
@@ -137,3 +137,20 @@ def rm_allow_noent(path):
         os.unlink(path)
     except FileNotFoundError:
         pass
+
+
+def import_ostree_commit(repo, commit, tarfile):
+    # create repo in case e.g. tmp/ was cleared out; idempotent
+    subprocess.check_call(['ostree', 'init', '--repo', repo, '--mode=archive'])
+
+    # in the common case where we're operating on a recent build, the OSTree
+    # commit should already be in the tmprepo
+    commitpartial = os.path.join(repo, f'state/{commit}.commitpartial')
+    if (subprocess.call(['ostree', 'show', '--repo', repo, commit]) == 0
```

Comment:
this spits out an ugly error to the screen if the commit doesn't exist. Can we use something like `subprocess.run(['ostree', 'show', '--repo', repo, commit], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL).returncode` ([new python API for subprocess](https://docs.python.org/3/library/subprocess.html#subprocess.call)) instead ?